schema/annotation-source.yaml

```diff
@@ -185,7 +190,23 @@ $defs:
         enum: [ "supports", "uncertain", "opposes" ]
         description: >-
           The direction of this statement with respect to the target proposition.
-
+      has_evidence_of_type: 
+        type: string
+        enum: [ "case-control evidence", "clinical testing evidence", "curation-based evidence", 
+        "author statement from literature", "phenotyping evidence", "provider interpretation evidence", 
+        "reference population evidence", "research evidence" ]
+        description: >-
+          A term describing a type of evidence used to assess the validity of Statement's proposition.          
+      statement_quality: 
```

Comment:
ClinVar review status is currently in a "clinvar-review-status" Extension,  but could be frame to fit into the VA `InformationEntity.information_quality` attribute (here, defined as a Statement-specific attribute that extends the base `informatoin_quality` attribute)